content/en/boilerplates/mc-env-var.md

```diff
@@ -0,0 +1,54 @@
+---
+---
+To simplify the deployment instructions and use the examples provided, set the
+following environment variables. If you use your own string values, ensure they
+are viable as filenames and compatible with `Makefiles`. Go to the [GNU guide](https://www.gnu.org/software/make/manual/html_node/Rule-Syntax.html)
+for more information.
+
+The guide assumes that the following environment variables are set:
+
+<table>
+   <thead>
+      <tr>
+      <th><strong>Environment Variable</strong></th>
+      <th><strong>Description</strong></th>
+      </tr>
+   </thead>
+      <tbody>
+      <tr>
+         <td>ISTIO</td>
+         <td>The top-level directory of the Istio installation.</td>
+      </tr>
+      <tr>
+         <td>CLUSTER_1</td>
+         <td>The name of the first cluster. The guide uses <code>cluster1</code>.</td>
+         </tr>
+      <tr>
+         <td>CLUSTER_2</td>
+         <td>The name of the second cluster. Must be different from the value in
+         <code>CLUSTER_1</code>. The guide uses <code>cluster2</code>.</td>
+         </tr>
+      <tr>
+         <td>CTX_1</td>
+         <td>The path to the Kubernetes context file for the first cluster.</td>
+      </tr>
+      <tr>
+         <td>CTX_2</td>
+         <td>The path to the Kubernetes context file for the second cluster.</td>
+      </tr>
+      <tr>
+         <td>NETWORK_1</td>
+         <td>The name of the first network. The guide uses <code>network1</code>.</td>
+      </tr>
+      <tr>
+         <td>MESH</td>
+         <td>The common mesh ID to use for all clusters in the mesh, for example
+         sample-mesh.</td>
+      </tr>
+      <tr>
+         <td>DISCOVERY_ADDRESS</td>
+         <td>Stores the IP address of the Istio ingress gateway of your first cluster,
```

Comment:
How do I get this? This is the install guide but it assumes its already installed? The doc never says "Step 1: install a single cluster" from what I can see

Reply:
@howardjohn This is not a document for beginners and the text expressly asks readers to be familiar with single cluster deployments already. Additionally, the content you see here is part of a boilerplate that can be reused, that's why you don't see any further guidance.

Reply:
Familiarity with single cluster != familiarity with knowing what a "network" is. I am fairly sure a large portion of Istio users would not understand what to set here.

And the doc says to *understand* single cluster, it doesn't say to deploy one.